fdbclient/include/fdbclient/EncryptKeyProxyInterface.h

```diff
@@ -176,7 +176,7 @@ struct EKPGetBaseCipherKeysByIdsRequest {
 
 	template <class Ar>
 	void serialize(Ar& ar) {
-		serializer(ar, arena, baseCipherInfos, debugId, reply);
+		serializer(ar, baseCipherInfos, debugId, reply, arena);
```

Comment:
for better understanding: putting `arena` as the last argument is a convention or a necessity?

Reply:
Putting the arena as the last argument is a necessity. If the arena isn't the last argument then it won't actually have shared ownership of memory referenced by *Ref members (StringRef, VectorRef, etc) for deserialized messages.